### PR TITLE
Align expanded

### PR DESCRIPTION
**Pull request by ChristianLieven**

### CODE REVIEW

File: src/views/Designs.vue [hunk 1]
```diff
@@ -64,142 +161,153 @@
...
+                            class="link"
+                            target="_blank"
+                          >
+                            {{ geneKnockout }}
+                          </a>
+                        </div>
+                        <div v-if="index >= 10" :hidden="!showAllGeneKnockouts">
+                          <a
+                            :href="
+                              `http://bigg.ucsd.edu/search?query=${geneKnockout}`
```
Comment (Midnighter) [inline src/views/Designs.vue]: As above.

File: src/views/Designs.vue [hunk 2]
```diff
@@ -64,142 +161,153 @@
...
-                        <a
-                          :href="
-                            reactionLink(reactionKnockin, design.method, true)
-                          "
-                          class="link"
-                          target="_blank"
+                        <div v-if="index < 10">
+                          <a
+                            :href="
+                              reactionLink(reactionKnockin, design.method, true)
```
Comment (Midnighter) [inline src/views/Designs.vue]: This is fine for now, in a future refactor I would make this an added property on the `reactionKnockin` itself.

File: src/views/Designs.vue [hunk 3]
```diff
@@ -64,142 +161,153 @@
...
-                          "
-                          class="link"
-                          target="_blank"
-                        >
-                          {{ geneKnockout }}
-                        </a>
+                        <div v-if="index < 10">
+                          <a
+                            :href="
+                              `http://bigg.ucsd.edu/search?query=${geneKnockout}`
```
[the remark this answers is not in the document]
Comment (kvikshaug) [reply src/views/Designs.vue]: The URL should be available in `settings.apis.bigg` too

**Comment by ChristianLieven** [comment]
> I couldn't see where this comes in but the styling of the table creates very wide margins between the table and the edge of the screen. This leads to a horizontal scrollbar appearing even though there is enough screen real estate to display the whole thing easily. Do you have any idea what could cause this?

The side margins are chiefly controlled by the v-flex size atrribute: `md10`. I think I've addressed the issue now by also setting `sm12` for the component.

File: src/views/Designs.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (Midnighter) [reply src/views/Designs.vue]: Actually, please disregard my comment. You're linking out to the actual BiGG frontend here and not fetching a resource. So as you have done it is perfectly fine.

**Pull request merged**